lib/node_modules/@stdlib/blas/base/drot/docs/repl.txt

```diff
@@ -0,0 +1,112 @@
+
+{{alias}}( N, x, strideX, y, strideY, c, s )
+    Applies a plane rotation.
+
+    The `N` and stride parameters determine how values
+    from `x` and `y` are rotated.
+
+    Indexing is relative to the first index. To introduce an offset, use typed
+    array views.
+
+    If `N` is less than or equal to `0`, the vectors are unchanged.
+
+    Parameters
+    ----------
+    N: integer
+        Number of indexed elements.
+
+    x: Float64Array
+        First input array.
+
+    strideX: integer
+        Index increment for `x`.
+
+    y: Float64Array
+        Second input array.
+
+    strideY: integer
+        Index increment for `y`.
+
+    c: float
+        Cosine of angle for rotation.
+
+    s: float
+        Sin of angle for rotation.
+
+    Returns
+    -------
+    y: Float64Array
+        Second input array.
+
+    Examples
+    --------
+    // Standard Usage:
+    > var x = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+    > var y = new {{alias:@stdlib/array/float64}}( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+    > {{alias}}( x.length, x, 1, y, 1, 1.0, 0.0 )
+    <Float64Array>[ 6.0, 7.0, 8.0, 9.0, 10.0 ]
```

Comment:
This is the input array. Values should change. Also, while `dswap` was emulated here, we should actually do

```
> {{alias}}( x.length, x, 1, y, 1, 1.0, 0.0 );
> x
<Float64Array>[ ??? ]
> y
<Float64Array>[ ??? ]
```

where `???` indicates to fill in the values.